modules/swagger-codegen/src/main/java/io/swagger/codegen/v3/examples/ExampleGenerator.java

```diff
@@ -251,7 +258,7 @@ private Object resolveModelToExample(String name, String mediaType, Schema schem
             return schema.getExample();
         }
         processedModels.add(name);
-        Map<String, Object> values = new HashMap<>();
+        Map<String, Object> values = new LinkedHashMap<>();
```

Comment:
This change ensures that the output order in the JSON is fixed, which makes testing easier.